src/test/java/LoginTests.java

```diff
@@ -1,25 +1,18 @@
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.chrome.ChromeOptions;
-import org.testng.Assert;
 import org.testng.annotations.Test;
-
-import java.time.Duration;
+import pages.HomePage;
+import pages.LoginPage;
 
 public class LoginTests extends BaseTest {
+    
     @Test
-    public void LoginEmptyEmailPassword() {
-
-//      Added ChromeOptions argument below to fix websocket error
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--remote-allow-origins=*");
+    public void LoginValidCredentialsTest() {
 
-        WebDriver driver = new ChromeDriver(options);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
 
-        String url = "https://testpro.io/";
-        driver.get(url);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-        driver.quit();
+        loginPage.provideEmail("camden.bertrand@testpro.io");
+        loginPage.providePassword("te$t$tudent");
+        loginPage.clickSubmitBtn();
```

Comment:
Please, add Assert

**One test - One Assert**

For example

![Clean_Test](https://github.com/CamdenBertrand/A49-QA-Automation-Framework/assets/104761482/9e1fb8e3-4c5c-45f3-8eec-9bfa87a81fc9)